### PR TITLE
Fix license info

### PR DESCRIPTION
**Pull request by hankjordan**
:-)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,10 +86,10 @@ and so on. But. But. Examples works like a charm. And this fact gives us a lot o
 
 ## License
 
-The `belly` is dual-licensed under either:
+`pecs` is dual-licensed under either:
```
Comment (jkb0o) [inline README.md:89]: Thank you! I fixed this one just before found your PR.

File: README.md [hunk 2]
```diff
@@ -86,10 +86,10 @@ and so on. But. But. Examples works like a charm. And this fact gives us a lot o
...
 
 - MIT License ([LICENSE-MIT](LICENSE-MIT) or [http://opensource.org/licenses/MIT](http://opensource.org/licenses/MIT))
 - Apache License, Version 2.0 ([LICENSE-APACHE](LICENSE-APACHE) or [http://www.apache.org/licenses/LICENSE-2.0](http://www.apache.org/licenses/LICENSE-2.0))
 
 This means you can select the license you prefer!
-This dual-licensing approach is the de-facto standard in the Rust ecosystem and there are [very good reasons](https://github.com/bevyengine/bevy/issues/2373) to include both.
\ No newline at end of file
+This dual-licensing approach is the de-facto standard in the Rust ecosystem and there are [very good reasons](https://github.com/bevyengine/bevy/issues/2373) to include both.
```
Comment (jkb0o) [inline README.md:95]: I can't figure out the difference. Is anything changed at this line?

Comment (hankjordan) [reply README.md:95]: No

**Comment by hankjordan** [comment]
Closed, you totally sniped me

**Pull request closed without merging**